### PR TITLE
Fix test problem

### PR DESCRIPTION
**Pull request by wuminzhe**

### CODE REVIEW

File: primitives/header-chain/tests/implementation_match.rs [hunk 1]
```diff
@@ -267,14 +270,15 @@ fn same_result_when_authority_equivocates_twice_in_a_round() {
...
+	assert_eq!(
 		finality_grandpa::validate_commit(
 			&justification.commit,
 			&full_voter_set(),
 			&AncestryChain::new(&justification.votes_ancestries),
 		)
-		.map(|result| result.ghost().cloned()),
-		Ok(Some(_))
+		.map(|result| result.is_valid()),
+		Ok(false)
```
Comment (fewensa) [inline primitives/header-chain/tests/implementation_match.rs:281]: I think there is `Ok(true)`

File: primitives/header-chain/tests/implementation_match.rs [hunk 2]
```diff
@@ -223,14 +225,15 @@ fn same_result_when_authority_equivocates_once_in_a_round() {
...
```
Comment (fewensa) [inline primitives/header-chain/tests/implementation_match.rs:236]: Same

File: primitives/header-chain/tests/implementation_match.rs [hunk 3]
```diff
@@ -189,14 +190,15 @@ fn same_result_when_justification_contains_duplicate_vote() {
...
```
Comment (fewensa) [inline primitives/header-chain/tests/implementation_match.rs:201]: Same

**Comment by fewensa** [comment]
https://github.com/paritytech/finality-grandpa/commit/36e409bcb76b41344be5b284732b529cd087432b#diff-b1a35a68f14e696205874893c07fd24fdb88882b47c23cc0e0c80a30c7d53759R511-R519

File: primitives/header-chain/tests/implementation_match.rs (hunk 1 above)
Comment (wuminzhe) [reply primitives/header-chain/tests/implementation_match.rs:281]: the is_valid() is more strict than checking ghost() is Some(_). assert will fail if true.

**Review by fewensa — APPROVED** [review]

**Pull request merged**